opsdroid/connector/matrix/connector.py

```diff
@@ -47,7 +45,7 @@ async def ensure_room_id(self, event):
             event.target = self.room_ids[event.target]
 
         if not event.target.startswith("!"):
-            event.target = await self.connection.get_room_id(event.target)
+            event.target = await self.connection.request_room_key(event.target)
```

Comment:
This isn't the right function. I think you need [`resolve_room_alias`](https://github.com/poljar/matrix-nio/blob/053be26ceca5a24422b471187f4d0e7d044807b5/nio/client/async_client.py#L1636)

Reply:
You're right